kombu/utils/json.py

```diff
@@ -29,7 +29,7 @@ class JSONEncoder(_encoder_cls):
     def default(self, o,
                 dates=(datetime.datetime, datetime.date),
                 times=(datetime.time,),
-                textual=(decimal.Decimal, uuid.UUID, DjangoPromise),
+                textual=(decimal.Decimal, DjangoPromise),
```

Comment:
should we document this change?

Reply:
Hmmmm, there's an entry for `- JSON serializer now handles datetimes, Django promise, UUID and Decimal.` in 4.0. It doesn't specify how it's handled but maybe (if accepted) this change could be documented as part of the next release?

Reply:
I pushed [6b89388](https://github.com/celery/kombu/pull/1575/commits/6b89388870271e23c31f6c2845f248c832fb7c6f) to keep the userguide (the only place I could find where JSON serialization is discussed) updated c: